utils/DeployedContractsHelpers.ts

```diff
@@ -255,6 +259,12 @@ export async function getBdx(hre: HardhatRuntimeEnvironment) {
   return (await hre.ethers.getContract("BDX", deployer)) as BDXShares;
 }
 
+export async function getTimelock(hre: HardhatRuntimeEnvironment): Promise<Timelock> {
+  const deployer = await getDeployer(hre);
+  const timelock = (await hre.ethers.getContract("Timelock", deployer)) as Timelock;
+  return timelock;
+}
+
```

Comment:
```suggestion
export async function getTimelock(hre: HardhatRuntimeEnvironment): Promise<Timelock> {
  const deployer = await getDeployer(hre);
  return hre.ethers.getContract("Timelock", deployer) as Timelock;
}
```

No need for the extra `await` here as all calling functions will use `await`